docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
I don't think we'll have the resources to maintain batch files in the long term. All functionality should be runnable via shell scripts and/or Makefiles.

Reply:
There is nothing to maintain here. Just like the `Makefile`, this is an auto-generated script that simply forwards all its arguments to the `sphinx-build` CLI. I agree that this file is of little use for us, but as it will require no maintenance and might be useful for other people, I am in favour of keeping it. 

If this file needs updating at any time, we will wait for a user to raise an issue and ask him/her to do a PR updating this file. 

What do you think of that?